### PR TITLE
Fix missing letter and add FAQ section to README

### PR DESCRIPTION
**Pull request by WasabiFan**

### CODE REVIEW

**Comment by WasabiFan** [comment]
I'm really tired, so to be honest I don't trust myself on even the simple things.

File: README.rst [hunk 1]
```diff
@@ -139,6 +141,10 @@ ev3python.com
     If you are just getting started with programming, we highly recommend
     that you check it out at `ev3python.com`!
 
+Frequently-Asked Questions
```
Comment (ddemidov) [inline README.rst]: Should this be a header? With `~~~~~~~` underline?

**Comment by ddemidov** [comment]
> I'm really tired, so to be honest I don't trust myself on even the simple things.

I can finish this myself if you like

File: README.rst (hunk 1 above)
Comment (WasabiFan) [inline README.rst]: It _should_ be part of the list thing that the other items are in... I have no idea what it's called :laughing:

**Comment by WasabiFan** [comment]
> I can finish this myself if you like

I'm happy to finish this up as long as it's just getting these changes ready.

File: README.rst (hunk 1 above)
Comment (ddemidov) [inline README.rst]: Ah, ok. Github helpfully folded the diff, so that I only saw `Using text-to-speech` above the changes.

**Comment by ddemidov** [comment]
Thanks!

**Pull request merged**